### PR TITLE
Fixes to ray-wand integration.

### PR DESCRIPTION
**Pull request by mvacaporale**
This PR adds `wandb` to the `requirements.txt` and adds check for `WANDB_DIR` variable in the `os.environ`. As well, there is added functionality to handle when the variable is not present.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -47,4 +47,5 @@ tabulate
 torchvision==0.5.0
 tqdm
 h5py
-pretrainedmodels==0.7.4
\ No newline at end of file
+pretrainedmodels==0.7.4
+wandb
```
Comment (lscheinkman) [inline requirements.txt:51]: Could you add the version you tested with?

Comment (mvacaporale) [reply requirements.txt]: You got it. Added.

File: nupic/research/frameworks/wandb/ray_wandb.py [hunk 2]
```diff
@@ -199,6 +211,9 @@ def get_latest_run_config():
 
 def get_latest_run_dir():
 
+    if not WANDB_DIR:
```
[the remark this answers is not in the document]
Comment (mvacaporale) [reply nupic/research/frameworks/wandb/ray_wandb.py]: Fixed.

File: nupic/research/frameworks/wandb/ray_wandb.py [hunk 3]
```diff
@@ -35,20 +36,27 @@
     "auto_init",
 ]
 
-# TODO: Add check for WANDB_DIR in `environ`
-WANDB_DIR = os.path.join(os.environ["WANDB_DIR"], "wandb")
+# Find directory of where wandb save it's results.
```
[the remark this answers is not in the document]
Comment (mvacaporale) [reply nupic/research/frameworks/wandb/ray_wandb.py]: Fixed.

**Pull request merged**